lasagne/init.py

```diff
@@ -141,6 +141,15 @@ class Glorot(Initializer):
        \\sigma &= \\sqrt{\\frac{2}{fan_{in}+fan_{out}}}\\\\
        W &\sim N(0, \\sigma)
 
+    For leaky ReLU ``gain`` can be calculated as
+
+    .. math::
+       gain = \\sqrt{\\frac{2}{1+\\alpha^2}}
+
+    where :math:`\\alpha` is the leakyness. For
+    ``leaky_rectify`` and
+    ``very_leaky_rectify`` :math:`\\alpha` is  0.01 and 0.333 respectively.
```

Comment:
Can you correctly cross-reference those? Should be:
```
:func:`leaky_rectify <lasagne.nonlinearities.leaky_rectify>`
```
or shorter (never tried that):
```
:func:`~lasagne.nonlinearities.leaky_rectify`
```